.templates/zigbee2mqtt/service.yml

```diff
@@ -1,10 +1,16 @@
 zigbee2mqtt:
   container_name: zigbee2mqtt
-  image: koenkk/zigbee2mqtt
+  build: ./.templates/zigbee2mqtt/.
+  environment:
+    - TZ=Etc/UTC
+  ports:
+    - "8080:8080"
```

Comment:
Just as an FYI, 8080 is a commonly used port. There maybe conflicts using this port with some setups.

Reply:
I did actually think about it. And I did note the potential for conflicts with

```
domoticz/service.yml:    - "8080:8080"
prometheus/service_cadvisor-arm.yml:      - 8080:8080
```

It just seemed to me that Zigbee2MQTT had had 8080:8080 defined before (albeit inactive due to host mode) but (at least until I zapped it in the earlier PR) still in a position to conflict with the above two were either of those defined at the same time.

In host mode it would grab 8080 so, had there been any conflicts we hadn't heard about, those must've been resolved by changing the others - yes? If, on the other hand, someone running Zigbee is unlikely to ever run those others (and I have no knowledge of this one way or the other; I'm just enumerating options) then it's a "no conflict".

Thus it seemed to me that keeping it on 8080 external was simply maintaining the status quo.

I have to admit to being slightly wary of changing ports **just** to avoid potential conflicts. Of all the things we do, I reckon slipping a port change under the radar sets up a whole bunch of users for failure once the template gets into their stack. Something like that can lurk for months.